content/api/templates.apib

```diff
@@ -404,39 +228,41 @@ For a master account owned template **only**, the results will include the `shar
-
-<banner status="info"><strong>Note</strong>: Attempting to update the published version of a template when only a draft version exists (and vice versa) will result in an error.</banner>
+### Update a Draft [PUT /templates/{id}]
 
-The `name` field may be modified, but the `id` field is read only.
+Updates the draft version of a template.
 
-If a content object is provided in the update request, it must
-contain all relevant content fields whether they are being changed or not.
+If a content object is provided in the request body, it must contain all relevant content fields whether they are being changed or not.
 The new content will completely overwrite the existing content.
 
-The example shows an update that will rename the template, enable open tracking,
-and update the content all in one API call. All content fields are included whether they are being
-changed or not.
++ Data Structure
+    + name (string) - Template name. Does not have to be unique. Maximum length - 1024 bytes
+    + description (string) - Description of the template. Maximum length - 1024 bytes
+    + content (object) - Content that will be used to construct an email. Substitution syntax is supported in all content attributes. Maximum length - 20 MBs
+        + html (string) - HTML content for the email's `text/html` MIME part. At a minimum, html or text is required. Expected in the UTF-8 charset with no `Content-Transfer-Encoding` applied.
+        + text (string) - Text content for the email's `text/plain` MIME part. At a minimum, html or text is required. Expected in the UTF-8 charset with no `Content-Transfer-Encoding` applied.
+        + subject (string, required) - Email subject line. Expected in the UTF-8 charset without RFC2047 encoding.
+        + from (enum, required) - Email address used to compose the email's `From` header. The domain must be a verified sending domain. Can be an object with `name` & `email` attributes: `{ "name": "My Company", "email": "deals@company.com" }`.
+            + (object)
+            + (string)
+        + reply_to (string) - Email address used to compose the email's `Reply-To` header.
+        + headers (object) - Object containing headers other than `Subject`, `From`, `To`, and `Reply-To`. See the [Header Notes](#header-header-notes).
+    + options (object) - Object containing template options. All template options can be overwritten in a transmission's options.
+        + transactional (boolean) - Distinguish between transactional and non-transactional messages for unsubscribe and suppression purposes.
+        + open_tracking (boolean) - Enable or disable open tracking.
+        + click_tracking (boolean) - Enable or disable click tracking.
+    + shared_with_subaccounts (boolean) - Whether this template can be used by subaccounts. Only present on templates that belong to a master account. Updating this will update it for all versions of the template.
```

Comment:
"Updating this will update it for all versions of the template." -> "Updating this will update both the published and draft versions."